.run/NotificationManagerApplication.run.xml

```diff
@@ -4,6 +4,7 @@
     <option name="FRAME_DEACTIVATION_UPDATE_POLICY" value="UpdateClassesAndResources" />
     <module name="qpay.notification-manager.main" />
     <option name="SPRING_BOOT_MAIN_CLASS" value="com.qpay.notificationmanager.NotificationManagerApplication" />
+    <option name="VM_PARAMETERS" value="-Dspring.mail.password=bmumqodhdzsvolkh" />
```

Comment:
Never commit your credentials. Right now, anyone in the world can stole this password and your email and use it.

In reality, developers use Kubernetes/Openshift or any other Cloud technologies to deploy apps and store there credentials. Since you don't use it, you have to manually each time add email and password to application.yaml and then don't commit it